modules/features/account/build.gradle

```diff
@@ -6,6 +6,7 @@ apply plugin: 'kotlin-kapt'
 android {
     namespace 'au.com.shiftyjelly.pocketcasts.account'
     buildFeatures {
+        buildConfig true
```

Comment:
I'm not sure if we need to access `BuildConfig` in feature module? If not, it should be disabled.

Reply:
```
* What went wrong:
A problem occurred configuring project ':modules:features:account'.
> defaultConfig contains custom BuildConfig fields, but the feature is disabled.
  To enable the feature, add the following to your module-level build.gradle:
  `android.buildFeatures.buildConfig true`
```

Yes it's necessary for this module.